x-pack/plugins/security_solution/public/resolver/view/panels/panel_content_utilities.tsx

```diff
@@ -6,10 +6,16 @@
 
 /* eslint-disable react/display-name */
 
-import { EuiCode } from '@elastic/eui';
+import { EuiCode, EuiToolTip, EuiPopover } from '@elastic/eui';
 import { i18n } from '@kbn/i18n';
 import styled from 'styled-components';
-import React, { memo } from 'react';
+import React, { memo, useState } from 'react';
+import { WithCopyToClipboard } from '../../../common/lib/clipboard/with_copy_to_clipboard';
```

Comment:
Is this from Kibana core?

Reply:
no, it utilizes an existing component within security_solution. I can spin up our own minimal version though?